packages/mdc-slider/README.md

```diff
@@ -143,8 +156,10 @@ instance that was affected.
 | `hasClass(className: string) => boolean` | Checks if `className` exists on the root element |
 | `addClass(className: string) => void` | Adds a class `className` to the root element |
 | `removeClass(className: string) => void` | Removes a class `className` from the root element |
-| `setThumbAttribute(name: string, value: string) => void` | Sets an attribute `name` to the value `value` on the thumb element. |
+| `setThumbAttribute(name: string, value: string) => void` | Sets an attribute `name` to the value `value` on the root element. |
+| `removeThumbAttribute(name: string) => void` | Removes an attribute `name` from the root element |
 | `computeBoundingRect() => ClientRect` | Computes and returns the bounding client rect for the root element. Our implementations calls `getBoundingClientRect()` for this. |
+| `getTabIndex() => number` | Returns the value of the `tabIndex` property on the thumb element |
```

Comment:
Rename this to `getThumbTabIndex` to follow the other thumb-specific naming.

Reply:
👍